test/Makefile

```diff
@@ -53,7 +53,7 @@ GINKGO_LABELS ?=
 # When --fail-fast is set, the entire suite will stop when the first failure occurs.
 # Enable --fail-fast by default.
 # https://onsi.github.io/ginkgo/#mental-model-how-ginkgo-handles-failure
-FAIL_FAST ?= true
+FAIL_FAST ?= false
```

Comment:
what is the impact for this change? will it take more time for testing?

Reply:
The `FAIL_FAST` flag determines how to handle errors met during the test suite running.
If `FAIL_FAST` is set to `true`, the test suite stops running immediately. 
If `FAIL_FAST` is set to `false`, the test suite continues running on meeting test error.

The `FAIL_FAST` flag is introduced to help debug the test error, so set the default value to false, and let the Jenkins pipeline can pass the value to determine whether the flag should be turned on.